Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp

```diff
@@ -130,8 +134,12 @@ void ThreadedCompositor::createGLContext()
     static_assert(sizeof(GLNativeWindowType) <= sizeof(uint64_t), "GLNativeWindowType must not be longer than 64 bits.");
     auto windowType = (GLNativeWindowType) m_nativeSurfaceHandle;
     m_context = GLContext::create(windowType, PlatformDisplay::sharedDisplayForCompositing());
+    m_eglSupportsDamage = false;
     if (m_context) {
         m_context->makeContextCurrent();
+        const char* extensions = eglQueryString(m_context->display().eglDisplay(), EGL_EXTENSIONS);
+        m_eglSupportsDamage = GLContext::isExtensionSupported(extensions, "EGL_KHR_swap_buffers_with_damage")
+            || GLContext::isExtensionSupported(extensions, "EGL_EXT_swap_buffers_with_damage");
```

Comment:
No need to check this again here, just query the display

Reply:
done